src/metabase/api/search.clj

```diff
@@ -558,12 +559,30 @@
                              :offset              mw.offset-paging/*offset*
                              :search-native-query search_native_query
                              :verified            verified}))
-        duration   (- (System/currentTimeMillis) start-time)]
-    ;; Only track global searches
-    (when (and (nil? models)
-               (nil? table_db_id)
-               (not archived))
-      (snowplow/track-event! ::snowplow/new-search-query api/*current-user-id* {:runtime-milliseconds duration}))
+        duration   (- (System/currentTimeMillis) start-time)
+        has-advanced-filters (some some?
+                                   [models created_by created_at last_edited_by
+                                    last_edited_at search_native_query verified])]
+    (when (not (nil? context))
+      (cond
+        (and
+         (nil? table_db_id)
+         (not archived))
```

Comment:
This is unconventional formatting and `when` should be used because there's only one branch
```suggestion
      (when (and (nil? table_db_id) (not archived))
```